### PR TITLE
Reducing explanation length

### PR DESCRIPTION
**Pull request by DhruvaBansal00**

### CODE REVIEW

File: src/autolabel/tasks/classification.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ClassificationTask(BaseTask):
...
 
-    GENERATE_EXPLANATION_PROMPT = "You are an expert at providing a well reasoned explanation for the output of a given task. \n\nBEGIN TASK DESCRIPTION\n{task_guidelines}\nEND TASK DESCRIPTION\nYou will be given an input example and the corresponding output. Your job is to provide an explanation for why the output is correct for the task above.\nThink step by step and generate an explanation. The last line of the explanation should be - So, the answer is <label>.\n{labeled_example}\nExplanation: "
+    GENERATE_EXPLANATION_PROMPT = "You are an expert at providing a well reasoned explanation for the output of a given task. \n\nBEGIN TASK DESCRIPTION\n{task_guidelines}\nEND TASK DESCRIPTION\nYou will be given an input example and the corresponding output. Your job is to provide an explanation for why the output is correct for the task above.\nThink step by step and generate an explanation with at most 3 to 5 sentences. The last line of the explanation should be - So, the answer is <label>.\n{labeled_example}\nExplanation: "
```
Comment (tuxracer) [inline src/autolabel/tasks/classification.py:34]: could we get a reasonable explanation in 1-2 sentences?

**Review by tuxracer — APPROVED** [review]

File: src/autolabel/tasks/classification.py (hunk 1 above)
Comment (nihit) [reply src/autolabel/tasks/classification.py:34]: yes, we should. Just limit it to <=2 sentences.

**Review by nihit — APPROVED** [review]
lgtm otherwise

File: src/autolabel/tasks/classification.py (hunk 1 above)
Comment (DhruvaBansal00) [reply src/autolabel/tasks/classification.py]: Limiting to 2 sentences now!

**Pull request merged**